### PR TITLE
Add SNAP explanation and Michigan Benefits section

### PR DESCRIPTION
**Pull request by rachelcope**
Reason for Change
===================
* Separate SNAP explanation from information about MichiganBenefits.
* Add info sectino on MichiganBenefits and privacy concerns
* https://trello.com/c/fO8I930e/225-home-as-a-user-i-want-a-clear-explanation-of-what-snap-is-and-how-this-site-helps

fireshot capture 39 - michigan benefits - http___localhost_3000_

### CODE REVIEW

File: app/assets/stylesheets/atoms/_layout.scss [hunk 1]
```diff
@@ -35,6 +35,10 @@
   }
 }
 
+.slab--centered {
+  text-align: center;
```
Comment (jessieay) [inline app/assets/stylesheets/atoms/_layout.scss]: there is a `text--centered` class that does this as well. would the atomic way of doing this be to apply that class to the div as well and not create another class?

File: app/views/pages/_home_page_bottom_section.html.erb [hunk 2]
```diff
@@ -24,3 +13,40 @@
...
+
+        <hr>
+
+        <p>
+          Private and Secure
+        </p>
+        <p class="text--help">
+          Your information is kept private and secure. MichiganBenefits
+          maintains strict security guidelines to protect your privacy and indentity.
+          For details, read the full <a href="/privacy" target="_blank">Privacy Policy</a>.
```
Comment (jessieay) [inline app/views/pages/_home_page_bottom_section.html.erb]: `For details, read the full <%= link_to "Privacy Policy", privacy_path, target: "_blank" %>.`

File: app/assets/stylesheets/atoms/_layout.scss (hunk 1 above)
Comment (rachelcope) [reply app/assets/stylesheets/atoms/_layout.scss]: @jessieay yes, I think it makes sense to use the text--centered class. I was referring to the GetCalFresh website that has a `slab--centered` class. But I like reusing the class that we already have. Will update

File: app/views/pages/_home_page_bottom_section.html.erb (hunk 2 above)
Comment (rachelcope) [reply app/views/pages/_home_page_bottom_section.html.erb]: 👍

**Pull request merged**